src/plugins/gitresolver/README.md

```diff
@@ -0,0 +1,23 @@
+- infos = Information about the gitresolver plugin is in keys below
+- infos/author = Name <name@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = resolver
+- infos/recommends =
+- infos/placements = rollback getresolver setresolver commit
+- infos/status = recommended productive maintained reviewed conformant compatible coverage specific unittest shelltest tested nodep libc configurable final preview 
+- infos/metadata =
+- infos/description =
+
+## Usage ##
+
+Copy this gitresolver if you want to start a new
```

Comment:
Extensive documentation ;)

Please give a short introduction and an example how to use it